src/pluginManager.ts

```diff
@@ -285,6 +285,7 @@ export class PluginManager {
 
             const absolutePath = path.join(searchPath, scopeDirectory);
             fs.readdirSync(absolutePath)
+              .filter(name => PluginManager.isQualifiedPluginIdentifier(name))
```

Comment:
Scoped packages can only be one level deep, this saves having to stat every directory in the @scope that cannot possibly be homebridge plugins.